Annotations/AddAnnotation/Properties/AssemblyInfo.cs

```diff
@@ -7,14 +7,7 @@
 // General Information about an assembly is controlled through the following
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("WpfPDFViewer")]
-[assembly: AssemblyDescription("")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("")]
-[assembly: AssemblyProduct("WpfPDFViewer")]
```

Comment:
These are the details for the output generated assembly. Why we are removing this?

Reply:
This assembly throw duplicate assembly error so removed this assembly.